features/in_onderzoek.feature

```diff
@@ -178,166 +178,234 @@ Functionaliteit: in onderzoek
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000280                       |
+      | fields              | geslacht,geboorte.datum         |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                       | waarde            |
+      | geslacht.code              | M                 |
+      | geslacht.omschrijving      | man               |
+      | geboorte.datum.type        | Datum             |
+      | geboorte.datum.langFormaat | 28 september 2001 |
+      | geboorte.datum.datum       | 2001-09-28        |
+
+    Scenario: persoon heeft voornamen in onderzoek dat niet is gevraagd met fields
+      Gegeven de persoon met burgerservicenummer '000000292' heeft de volgende gegevens
+      | naam                            | waarde     |
+      | aanduiding in onderzoek (83.10) | 010210     | 
+      | datum ingang onderzoek (83.20)  | 20220307   |
+      | geslachtsnaam (02.40)           | Zonderling |
+      | geboortedatum (03.10)           | 20010928   |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000292                       |
+      | fields              | geboorte,naam.geslachtsnaam     |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                       | waarde            |
+      | naam.geslachtsnaam         | Zonderling        |
+      | geboorte.datum.type        | Datum             |
+      | geboorte.datum.langFormaat | 28 september 2001 |
+      | geboorte.datum.datum       | 2001-09-28        |
 
-  @proxy
   Rule: wanneer een element in de bron in onderzoek is, wordt het gegeven in het antwoord dat daaruit gevuld wordt ook in inOnderzoek opgenomen met de waarde true
 
-    Abstract Scenario: persoon heeft <veld> in onderzoek
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                        | waarde    |
-      | burgerservicenummer         | 555550001 |
-      En de persoon heeft de volgende 'persoonInOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | <waarde> |
-      | datumIngangOnderzoek          | 20220307 |
-      Als personen wordt gezocht met de volgende parameters
+    Abstract Scenario: persoon heeft <in onderzoek> in onderzoek
+      Gegeven de persoon met burgerservicenummer '000000309' heeft de volgende gegevens
+      | naam                            | waarde                    |
+      | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+      | datum ingang onderzoek (83.20)  | 20220307                  |
+      | geslachtsaanduiding (04.10)     | M                         |
+      Als gba personen wordt gezocht met de volgende parameters
```

Comment:
moet hier niet 'personen' staan ipv 'gba personen'?

Reply:
Klopt, is gecorrigeerd.